DigitalLearningSolutions.Data/Models/CourseCompletion/CourseCompletion.cs

```diff
@@ -0,0 +1,58 @@
+﻿namespace DigitalLearningSolutions.Data.Models.CourseCompletion
+{
+    using System;
+
+    public class CourseCompletion
+    {
+        public int Id { get; }
+        public string ApplicationName { get; }
+        public string CustomisationName { get; }

```

Comment:
Should the two fields above be combined to a Title fields, as we have done for other models? Or is there a reason to keep them separate? 🙂

Reply:
No reason, I had just forgotten about that! Changed in 2cbdd33 🙂